### ISSUE TITLE
Content rendering error - MDN JavaScript docs 

### ISSUE THREAD

**Issue by taulant** [zealdocs/zeal#1321]
**Steps to reproduce**
1. Download MDN JavaScript docs
2. Search for a JavaScript related article on Zeal (ex. Boolean)
3. You can see the result for the article in the left bar
4. When you open the article you can only see the title and not the content

![image](https://user-images.githubusercontent.com/1487469/107838834-c9d96180-6d75-11eb-8649-bdf7bdea0db8.png)

PS. The issue is reducible with other MDN docs like CSS and HTML

**Comment by marvka** [comment]
Duplicate of #1155 . It's already fixed, just not in the current release version. Will be in 0.7.0, which will hopefully come out sometime soon. You can build it from source in the meantime or grab the CI version! :)

**Comment by David-Else** [comment]
@marvka 
> ...or grab the CI version! :)

How can I download the CI appimage binary?! I can't seem to find a link anywhere in the Github Actions area. Thanks!

**Comment by atleta** [comment]
@David-Else you can indeed fix this by removing a JS file from the docsets that cause a problem. See: https://github.com/zealdocs/zeal/issues/1155#issuecomment-552956765

**Comment by David-Else** [comment]
@atleta Cheers, I knew about that, just want the working new version now, no more hacks.

**Comment by atleta** [comment]
@David-Else Ah, OK. I just commented because I was happy to find this solution *again*. Because apparently I did know about it, but forgot it and to me installing an unstable build (esp. when through the package manager) feels less of a solution.

As far as I can see that issue has been open for well over a year and the last release is almost 2.5 years old. I don't get why they don't do releases more often.
